README.md

```diff
@@ -119,6 +119,7 @@ Next we have to customize the webhook payload, as we want change some of the dat
 4. Scroll down the bottom of the webhook configuration screen and in "Payload" section replace `{{{JSONstringify this}}}` with the following:
 
 ```handlebars
+{{#withDeliveryContentItem contentItemId=payload.rootContentItem.id account="account-id"}}
```

Comment:
For a snapshot published webhook it will also have teh snapshotId param
`{{#withDeliveryContentItem contentItemId=payload.rootContentItem.id account="ampproduct" stagingEnvironment="account-id" snapshotId=payload.id}}
`